docs/preview/02-Features/writing-different-telemetry-types.md

```diff
@@ -388,6 +389,33 @@ catch (Exception exception)
 }
 ```
 
+### Making it easier to link services
+
+Service-to-service correlation requires linkage between tracked dependencies (outgoing) and requests (incoming).
+Tracking any kind of dependency with the library has the possibility to provide an dependency ID. This ID will be needed later when the incoming request is tracked (dependency ID = request's parent ID).
```

Comment:
```suggestion
Tracking any kind of dependency with the library has the possibility to provide a dependency ID. This dependency ID will be needed later when the incoming request is tracked (dependency ID = request's parent ID).
```

Reply:
I'm not sure how to interpret the last sentence ? The (dependency) ID is needed later when the incoming request is tracked.   You mean that the dependencyId is used in the logging of the downstream dependency to correlate the request id with the dependency id ?

Reply:
Yes, something like that. Maybe we should specify that the 'later' part is in the other dependency.

Reply:
I updated the last sentence with a more thorough explanation. We also need to update the links but that can only be done when #250 is done.